<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="informationTable">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>421</width>
      <height>192</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <column>
     <property name="text">
      <string>№</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип объекта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип орбиты</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Гос.принадлежит</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="buttonRevert">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>230</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить действие</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="infoTLE">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>420</y>
      <width>271</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>TLE</string>
    </property>
    <widget class="QLabel" name="labelLatitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Широта</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLongitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Долгота</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAltitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Высота</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelVelocity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Скорость</string>
     </property>
    </widget>
    <widget class="QLabel" name="valueLatitude">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valueLongitude">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valueAltitude">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valueVelocity">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="fileSelect">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>270</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл для загрузки TLE</string>
    </property>
   </widget>
   <widget class="MyGraphicsView" name="fieldDraw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>680</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="timeSrtup">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>422</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Дата</string>
    </property>
    <widget class="QSpinBox" name="boxDay">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string>00.00.0000 | 00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="boxMonth">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="boxYear">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1950</number>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="boxHour">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="boxMinute">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QPushButton" name="setupTime">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Установить</string>
     </property>
    </widget>
    <widget class="QSlider" name="changeTime">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>144</number>
     </property>
     <property name="sliderPosition">
      <number>72</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelminus3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>-3дня</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelplus3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>+3 дня</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentValue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelDay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>62</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>День:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMonth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Месяц:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelYear">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Год:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelHour">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Часы:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinute">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Минуты:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="orderBox">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>№</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Название</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Тип объекта</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Тип орбиты</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Гос. принадлежность</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>80</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="orderBoxType">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>60</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>В обратном порядке</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButtonAllTrace">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отрисовать все трассы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButtonInternet">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить из Интернета</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1343</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview_2.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
